packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.routing.js

```diff
@@ -120,9 +90,9 @@ export default /* @ngInject */ ($stateProvider) => {
         }),
 
       trackPage: /* @ngInject */ (atInternet) => (hit) =>
-        atInternet.trackPage({
+        atInternet.trackClick({
           name: `${COLD_ARCHIVE_TRACKING_PREFIX}::${hit}`,
-          type: 'action',
+          type: 'page',
```

Comment:
**nitpick:** `trackClick` and `type:page` does not make a good combination.
Please refer https://www.npmjs.com/package/ng-at-internet/v/3.1.1#click-tracking

Reply:
I guess the current implementation is the right one:
if we refer to ng-at-internet module w have those possibilities:
`
atInternet.trackClick({
    name: "my-click", // name of action
    type: "action",   // event type (optional) Possible values : action/navigation/download/exit
    level2: "1",      // (optional) if not configured in defaults you must specify your project id
    visitorId: "123"  // (optional) set identifiedVisitor.id sent with each hit
});
`

Reply:
Completed